src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -208,7 +212,6 @@ class RtpReceiverImpl @JvmOverloads constructor(
                         node(srtcpDecryptWrapper)
                         node(toggleablePcapWriter.newObserverNode())
                         node(CompoundRtcpParser(logger))
-                        node(silenceDiscarder.rtcpNode)
```

Comment:
Why don't we need this anymore?

Reply:
It was only doing timestamp rewriting, and we don't have the code to do that anymore.